pages/3_Compare_Cases.py

```diff
@@ -1,26 +1,79 @@
 # pylint: disable=invalid-name
 
+import extra_streamlit_components as stx
 import streamlit as st
 
 from sibylapp.compute.context import get_term
 from sibylapp.view import entity_difference
 from sibylapp.view.utils import filtering, setup
 
-setup.setup_page()
+setup.setup_page(return_row_ids=True)
+
+
+tab = stx.tab_bar(
+    data=[
+        stx.TabBarItemData(
+            id=1, title=f"Difference within single {get_term('entity')}", description=""
```

Comment:
You need to capitalize 'Entity' to make this get_term call work correctly; you can then set lower to true. ie `get_term('Entity', lower=True)`. Not ideal I know, something we can try to change in the future